libs/shared/map-viewer/map-viewer.component.scss

```
@@ -0,0 +1,151 @@
+}
+
+.map-label {
+  margin: 0;
+  font-size: 24px;
+  font-weight: 600;
+  text-align: center;
+}
+
+.map-image-container {
+  position: relative;
+  // Inline preview size; clicking opens the full map in a dialog.
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)